src/features/page_view_timing/aggregate/index.js

```diff
@@ -66,7 +66,7 @@ export class Aggregate extends AggregateBase {
 
     /* First Input Delay (+"First Interaction") - As of WV v3, it still imperfectly tries to detect document vis state asap and isn't supposed to report if page starts hidden. */
     onFID(({ name, value, entries }) => {
-      if (pageStartedHidden || this.alreadySent.has(name)) return
+      if (pageStartedHidden || this.alreadySent.has(name) || !Array.isArray(entries) || entries.length === 0) return
```

Comment:
```suggestion
      if (pageStartedHidden || this.alreadySent.has(name) || entries.length === 0) return
```
entries is always guaranteed to be `[]` at least